.github/workflows/ci-scripts/ubuntu.sh

```diff
@@ -12,14 +12,29 @@ echo "CFG: $CFG"
 echo "LTO: $LTO"
 echo "CACHE_DIR: $CACHE_DIR"
 
-echo FROM wesnoth/wesnoth:"$IMAGE"-"$BRANCH" > utils/dockerbuilds/CI/Dockerfile-CI-"$IMAGE"-"$BRANCH"
-echo COPY ./ /home/wesnoth-CI/ >> utils/dockerbuilds/CI/Dockerfile-CI-"$IMAGE"-"$BRANCH"
-echo WORKDIR /home/wesnoth-CI >> utils/dockerbuilds/CI/Dockerfile-CI-"$IMAGE"-"$BRANCH"
+if [ "$IMAGE" == "steamrt" ]; then
+		cd utils/dockerbuilds/
+		./make_steam_build || exit 1
+		tar -cf steambuild.tar steambuild
+		mv steambuild.tar ~/steambuild.tar
+elif [ "$IMAGE" == "mingw" ]; then
+		git archive --format=tar HEAD > wesnoth.tar
+		mv wesnoth.tar ~/wesnoth.tar
+		cd utils/dockerbuilds/
+		./make_mingw_build || exit 1
+		cd mingwbuild
+		mv $(find . -maxdepth 1 -regex "./wesnoth.*-win64.exe") ~/wesnoth-installer.exe
```

Comment:
This should be written as:
```suggestion
		mv ./wesnoth*-win64.exe ~/wesnoth-installer.exe
```